src/BloomBrowserUI/pageChooser/blank-preview.css

```diff
@@ -0,0 +1,35 @@
+/*[fmt]1A90-1A0A-1*/
+/*blank-preview.css - rules for the blank preview page for the Bloom Page Chooser dialog*/
+* {
+	position: relative;
+	margin: 0;
+	padding: 0;
+}
+BODY {
+	font-size: 100%;
+	font-family: sans-serif;
```

Comment:
Perhaps it's badly named. It displays a message letting the user know that a preview will show up here when they select a page.